Purpose: Internationalise a few more messages

Fixes: #266281

Status wrt upstream: submitted 20050613 (slightly differently as upstream 
                                         differs on some files)

Index: shadow-4.0.12/libmisc/obscure.c
===================================================================
--- shadow-4.0.12.orig/libmisc/obscure.c	2005-08-17 11:49:14.000000000 +0200
+++ shadow-4.0.12/libmisc/obscure.c	2005-08-17 11:51:09.000000000 +0200
@@ -158,7 +158,7 @@
 #endif
 
 	if (strcmp (new, old) == 0)
-		return "no change";
+		return _("no change");
 
 	newmono = str_lower (xstrdup (new));
 	oldmono = str_lower (xstrdup (old));
@@ -167,19 +167,19 @@
 	strcat (wrapped, oldmono);
 
 	if (palindrome (oldmono, newmono))
-		msg = "a palindrome";
+		msg = _("a palindrome");
 
 	if (!msg && strcmp (oldmono, newmono) == 0)
-		msg = "case changes only";
+		msg = _("case changes only");
 
 	if (!msg && similar (oldmono, newmono))
-		msg = "too similar";
+		msg = _("too similar");
 
 	if (!msg && simple (old, new))
-		msg = "too simple";
+		msg = _("too simple");
 
 	if (!msg && strstr (wrapped, newmono))
-		msg = "rotated";
+		msg = _("rotated");
 
 #ifdef HAVE_LIBCRACK
 	/*
@@ -215,7 +215,7 @@
 	newlen = strlen (new);
 
 	if (newlen < getdef_num ("PASS_MIN_LEN", 0))
-		return "too short";
+		return _("too short");
 
 	/*
 	 * Remaining checks are optional.
Index: shadow-4.0.12/libmisc/utmp.c
===================================================================
--- shadow-4.0.12.orig/libmisc/utmp.c	2005-08-17 11:49:14.000000000 +0200
+++ shadow-4.0.12/libmisc/utmp.c	2005-08-17 11:51:09.000000000 +0200
@@ -53,9 +53,9 @@
 extern void endutent ();
 
 #define	NO_UTENT \
-	"No utmp entry.  You must exec \"login\" from the lowest level \"sh\""
+	_("No utmp entry.  You must exec \"login\" from the lowest level \"sh\"")
 #define	NO_TTY \
-	"Unable to determine your tty name."
+	_("Unable to determine your tty name.")
 
 /*
  * checkutmp - see if utmp file is correct for this process
Index: shadow-4.0.12/src/login.c
===================================================================
--- shadow-4.0.12.orig/src/login.c	2005-08-17 11:50:56.000000000 +0200
+++ shadow-4.0.12/src/login.c	2005-08-17 11:51:09.000000000 +0200
@@ -595,7 +595,7 @@
 		retcode = pam_start ("login", username, &conv, &pamh);
 		if (retcode != PAM_SUCCESS) {
 			fprintf (stderr,
-				 "login: PAM Failure, aborting: %s\n",
+				 _("login: PAM Failure, aborting: %s\n"),
 				 pam_strerror (pamh, retcode));
 			SYSLOG ((LOG_ERR, "Couldn't initialize PAM: %s",
 				pam_strerror (pamh, retcode)));
@@ -713,7 +713,7 @@
 			  if (!failed)
 			    break;
 			  
-			  fprintf(stderr,"Login incorrect\n\n");
+			  fprintf(stderr,_("Login incorrect\n\n"));
 			  if (pwd && getdef_bool("FAILLOG_ENAB"))
 			    failure (pwent.pw_uid, tty, &faillog);
 			  if (getdef_str("FTMP_FILE") != NULL) {
@@ -1018,7 +1018,7 @@
 	child = fork ();
 	if (child < 0) {
 		/* error in fork() */
-		fprintf (stderr, "login: failure forking: %s",
+		fprintf (stderr, _("login: failure forking: %s"),
 			 strerror (errno));
 		PAM_END;
 		exit (0);
Index: shadow-4.0.12/src/logoutd.c
===================================================================
--- shadow-4.0.12.orig/src/logoutd.c	2005-08-17 11:49:14.000000000 +0200
+++ shadow-4.0.12/src/logoutd.c	2005-08-17 11:51:09.000000000 +0200
@@ -41,7 +41,7 @@
 static char *Prog;
 
 #ifndef DEFAULT_HUP_MESG
-#define DEFAULT_HUP_MESG "login time exceeded\r\n"
+#define DEFAULT_HUP_MESG _("login time exceeded\r\n")
 #endif
 
 #ifndef HUP_MESG_FILE
Index: shadow-4.0.12/src/newgrp.c
===================================================================
--- shadow-4.0.12.orig/src/newgrp.c	2005-08-17 11:49:14.000000000 +0200
+++ shadow-4.0.12/src/newgrp.c	2005-08-17 11:51:09.000000000 +0200
@@ -467,7 +467,7 @@
 		child = fork ();
 		if (child < 0) {
 			/* error in fork() */
-			fprintf (stderr, "%s: failure forking: %s",
+			fprintf (stderr, _("%s: failure forking: %s"),
 				 is_newgrp ? "newgrp" : "sg", strerror (errno));
 			exit (1);
 		} else if (child) {
Index: shadow-4.0.12/src/su.c
===================================================================
--- shadow-4.0.12.orig/src/su.c	2005-08-17 11:50:57.000000000 +0200
+++ shadow-4.0.12/src/su.c	2005-08-17 11:51:09.000000000 +0200
@@ -329,7 +329,7 @@
 	if (ret != PAM_SUCCESS) {
 		SYSLOG ((LOG_ERR, "pam_close_session: %s",
 			 pam_strerror (pamh, ret)));
-		fprintf (stderr, "%s: %s\n", Prog, pam_strerror (pamh, ret));
+		fprintf (stderr, _("%s: %s\n"), Prog, pam_strerror (pamh, ret));
 		pam_end (pamh, ret);
 		exit (1);
 	}
@@ -526,7 +526,7 @@
 	if (ret != PAM_SUCCESS) {
 		SYSLOG ((LOG_ERR, "pam_set_item: %s",
 			 pam_strerror (pamh, ret)));
-		fprintf (stderr, "%s: %s\n", Prog, pam_strerror (pamh, ret));
+		fprintf (stderr, _("%s: %s\n"), Prog, pam_strerror (pamh, ret));
 		pam_end (pamh, ret);
 		exit (1);
 	}
@@ -649,7 +649,7 @@
 	if (ret != PAM_SUCCESS) {
 		SYSLOG ((LOG_ERR, "pam_authenticate: %s",
 			 pam_strerror (pamh, ret)));
-		fprintf (stderr, "%s: %s\n", Prog, pam_strerror (pamh, ret));
+		fprintf (stderr, _("%s: %s\n"), Prog, pam_strerror (pamh, ret));
 		pam_end (pamh, ret);
 		su_failure (tty);
 	}
@@ -662,7 +662,7 @@
 		} else {
 			SYSLOG ((LOG_ERR, "pam_acct_mgmt: %s",
 				 pam_strerror (pamh, ret)));
-			fprintf (stderr, "%s: %s\n", Prog,
+			fprintf (stderr, _("%s: %s\n"), Prog,
 				 pam_strerror (pamh, ret));
 			pam_end (pamh, ret);
 			su_failure (tty);
@@ -798,7 +798,7 @@
 	ret = pam_setcred (pamh, PAM_ESTABLISH_CRED);
 	if (ret != PAM_SUCCESS) {
 		SYSLOG ((LOG_ERR, "pam_setcred: %s", pam_strerror (pamh, ret)));
-		fprintf (stderr, "%s: %s\n", Prog, pam_strerror (pamh, ret));
+		fprintf (stderr, _("%s: %s\n"), Prog, pam_strerror (pamh, ret));
 		pam_end (pamh, ret);
 		exit (1);
 	}
@@ -807,7 +807,7 @@
 	if (ret != PAM_SUCCESS) {
 		SYSLOG ((LOG_ERR, "pam_open_session: %s",
 			 pam_strerror (pamh, ret)));
-		fprintf (stderr, "%s: %s\n", Prog, pam_strerror (pamh, ret));
+		fprintf (stderr, _("%s: %s\n"), Prog, pam_strerror (pamh, ret));
 		pam_setcred(pamh, PAM_DELETE_CRED);
 		pam_end (pamh, ret);
 		exit (1);
Index: shadow-4.0.12/src/useradd.c
===================================================================
--- shadow-4.0.12.orig/src/useradd.c	2005-08-17 11:51:02.000000000 +0200
+++ shadow-4.0.12/src/useradd.c	2005-08-17 11:51:09.000000000 +0200
@@ -790,7 +790,7 @@
 		ngrp->gr_mem = add_list (ngrp->gr_mem, user_name);
 		if (!gr_update (ngrp)) {
 			fprintf (stderr,
-				 "%s: error adding new group entry\n", Prog);
+				 _("%s: error adding new group entry\n"), Prog);
 			fail_exit (E_GRP_UPDATE);
 		}
 		SYSLOG ((LOG_INFO, "add `%s' to group `%s'",
Index: shadow-4.0.12/src/chage.c
===================================================================
--- shadow-4.0.12.orig/src/chage.c	2005-08-17 11:51:02.000000000 +0200
+++ shadow-4.0.12/src/chage.c	2005-08-17 11:51:09.000000000 +0200
@@ -563,7 +563,7 @@
 	}
 
 	if (lflg && (setgid (getgid ()) || setuid (ruid))) {
-		fprintf (stderr, "%s: failed to drop privileges (%s)\n",
+		fprintf (stderr, _("%s: failed to drop privileges (%s)\n"),
 			 Prog, strerror (errno));
 		exit (E_NOPERM);
 	}
